### ISSUE TITLE
too many features using corrector using api.py

### ISSUE THREAD

**Issue by chaosido** [jxmorris12/vec2text#79]
So when we create the dataset in our reproduction scripts we have the following features:

Dataset({
    features: ['input_ids', 'attention_mask', 'embedder_input_ids', 'embedder_attention_mask', 'labels'],
    num_rows: 256
})

But then when you call corrector.evaluate, the  custom collator function (see collator.py) receives only the label feature. We don’t know if we should change somehow, but we suspect somewhere in the corrector.evaluate (the evaluation fn comes from the transformers package) all  the other features are dropped
